app/Providers/ModelServiceProvider.php

```diff
@@ -37,6 +39,22 @@ public function boot()
                 'content' => $review->toJson(),
             ]);
         });
+
+        Tagged::saved(function ($tagged) {
+            $post = Post::find($tagged->taggable_id);
+            $adminId = auth()->user()->northstar_id;
+
+            $post->action = [
+                'type' => 'tagged',
+                'admin' => $adminId,
+            ];
+
+            Event::create([
+                'eventable_id' => $post->id,
+                'eventable_type' => $post->getModel(),
```

Comment:
@sbsmith86 changed it to this since it seems a little safer to do but if it's better to hardcode this, let me know!